features/index_page.feature

```diff
@@ -0,0 +1,15 @@
+Feature: index page with a list restaurants
+  As a visitor
+  in order to access the app
+  I need to see an index page
+  I would like to be presented with some examples of restaurants that deliver food
+
+ Background:
+ Given the following restaurant exists
+   | name        | street      | zip_code | city     | food_style                |
+   | Denmark     | Kyra Green  | 0        | Pagacside| Iran, Islamic Republic of |
```

Comment:
Also, since you are not using most of the attributes for assertions I would allow the factory to set them. Not needed in the table